src/blame.c

```diff
@@ -133,8 +133,11 @@ git_blame* git_blame__alloc(
 		return NULL;
 	}
 
-	if (opts.flags & GIT_BLAME_USE_MAILMAP)
-		git_mailmap_from_repository(&gbr->mailmap, repo);
+	if (opts.flags & GIT_BLAME_USE_MAILMAP &&
+		git_mailmap_from_repository(&gbr->mailmap, repo) < 0) {
```

Comment:
No need to change that now, but our agreed-upon code style is to indent continuation lines in conditionals with four spaces:

```
if (opts.flags & GIT_BLAME_USE_MAILMAP &&
    git_mailmap_from_repository(&gbr->mailmap, repo) < 0) {
        git_blame_free(gbr);
        return NULL;
}
```